kernel/arm64/KERNEL.TSV110

```diff
@@ -0,0 +1,175 @@
+SAMINKERNEL  = ../arm/amin.c
```

Comment:
The file could be greatly shortened by including ARMV8 kernel and leaving just optimized overrides here. What do you think?

Reply:
Seems to me the recent rewrites by @rengolin did away with the "include KERNEL.ARMV8" , so the current redundancy would be acceptable. (In particular if this is to be seen as initial cpu support to be improved upon later).
Perhaps some rainy afternoon someday could be spent rearranging all the arm64 KERNEL files so that they all list the kernels in the same order for easier comparison :slightly_smiling_face:

Reply:
Yes, it is good, CPUID would be handy, otherwise complete enough

Reply:
The current file is just adding new platform support. This file is not the final version and will be continuously optimized and modified.